### DISCUSSION TITLE
scrolling notes in mobile

### DISCUSSION THREAD

**Discussion by jhc86** [TriliumNext/Trilium#1768] [Q&A] [answered]
I've tried a couple Android devices and Firefox 86.1.1 will display note content in a popup that doesn't scroll when swiped. I've tried double finger swiping, and if I try to edit the note, I can move the cursor off the page and the edit will register OK and is seen on desktop side.

Does anyone know a trick to scroll the view of note content in mobile mode?

**Comment by zadam** [comment, marked as answer]
Hi, this was a bug, I've just fixed that and will release it in the next stable version. Thanks for reporting this!
